gflownet/envs/base.py

```diff
@@ -20,40 +21,43 @@ class GFlowNetEnv:
 
     def __init__(
         self,
-        device="cpu",
-        float_precision=32,
-        env_id=None,
-        reward_beta=1,
-        reward_norm=1.0,
-        reward_norm_std_mult=0,
-        reward_func="power",
-        energies_stats=None,
-        denorm_proxy=False,
+        device: str = "cpu",
+        float_precision: int = 32,
+        env_id: Union[int, str] = "env",
+        reward_min: float = 1e-8,
+        reward_beta: float = 1.0,
+        reward_norm: float = 1.0,
+        reward_norm_std_mult: float = 0.0,
+        reward_func: str = "identity",
+        energies_stats: List[int] = None,
+        denorm_proxy: bool = False,
         proxy=None,
         oracle=None,
-        proxy_state_format=None,
+        proxy_state_format: str = "oracle",
+        fixed_distribution: dict = None,
```

Comment:
I'm afraid these are not optional in these continuous environments because they are used by [`get_policy_output()`](https://github.com/alexhernandezgarcia/gflownet/blob/tests-and-simplify-envs/gflownet/envs/htorus.py#L103).

Reply:
Does that make them optional or not? Is there a conditionally optional? I guess this will be solved with separate continuous and discrete base envs as @AlexandraVolokhova has suggested for other reasons.

Reply:
I think that makes them optional in general, and ideally there would be a typing change for the continuous envs (or at least a check in the init, with exception raised if it's None in that context). Anyhow, I think `var_name: dict = None` is wrong either way, and it should either not have a None as default and be required, or have it and be optional. But I am fine with leaving this as is it here is supposed to be continuous base env soon.